src/App.tsx

```diff
@@ -1,14 +1,45 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { ErrorNotification } from './components/ErrorNotification';
+import { ErrorType, FilterType } from './types/ErrorType';
+import cn from 'classnames';
+import { getFilteredTodos } from './utils/getFilteredTodos';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<ErrorType>(
+    ErrorType.DEFAULT,
+  );
+  const [loading, setLoading] = useState(false);
```

Comment:
boolean should start with is has
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```